python/ray/data/dataset.py

```diff
@@ -3252,16 +3272,24 @@ def iter_torch_batches(
                 The final batch may include fewer than ``batch_size`` rows if
                 ``drop_last`` is ``False``. Defaults to 256.
             dtypes: The Torch dtype(s) for the created tensor(s); if ``None``, the dtype
-                is inferred from the tensor data.
-            device: The device on which the tensor should be placed; if ``None``, the
-                torch tensor is constructed on the CPU.
+                is inferred from the tensor data. This parameter cannot be used in
+                conjunction with ``collate_fn``.
+            device: The device on which the tensor should be placed. Defaults to
+                "auto" which moves the tensors to the appropriate device when the
+                Dataset is passed to Ray Train and ``collate_fn`` is not provided.
+                Otherwise, defaults to CPU. This parameter cannot be used in
+                conjunction with ``collate_fn``.
```

Comment:
```suggestion
            device: The device on which the tensor should be placed. Defaults to
                ``"auto"`` which moves the tensors to the appropriate device when the
                :class:`~ray.data.Dataset` is passed to Ray Train and ``collate_fn`` isn't provided.
                Otherwise, defaults to CPU. You can't use this parameter with ``collate_fn``.
```